openpype/client/entities.py

```diff
@@ -81,6 +94,20 @@ def get_projects(active=True, inactive=False, fields=None):
 
 
 def get_project(project_name, active=True, inactive=True, fields=None):
+    """Return project entity document by project name.
+
+    Args:
+        project_name (str): Name of project.
+        active (Optional[bool]): Allow active project. Defaults to True.
+        inactive (Optional[bool]): Allow inactive project. Defaults to True.
+        fields (Iterable[str]): Fields that should be returned. All fields are
+            returned if 'None' is passed.
```

Comment:
Should this be `Optional[Iterable[str]]` @iLLiCiTiT 

```suggestion
        fields (Optional[Iterable[str]]): Fields that should be returned.
            All fields are returned if 'None' is passed.
```

Reply:
Yes

Reply:
Done.